com.microsoft.mrtk.input/Assets/Prefabs/MRTK RightHand Controller.prefab

```diff
@@ -107,6 +115,54 @@ PrefabInstance:
       propertyPath: handNode
       value: 5
       objectReference: {fileID: 0}
+    - target: {fileID: 2572330550550829917, guid: a9ae130324a36134e99beedf50e7be6f, type: 3}
+      propertyPath: pinchPoseSource.hand
+      value: 2
+      objectReference: {fileID: 0}
+    - target: {fileID: 2940030942784507886, guid: a9ae130324a36134e99beedf50e7be6f, type: 3}
+      propertyPath: fallbackPalmPoseSource.hand
```

Comment:
Still looks like there's some stale serialization in the overrides....you might have to rebuild these changes to remove it

Reply:
Stripped out these outdated property references!